src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.edifact.dat

```diff
@@ -0,0 +1,18 @@
+UNB+UNOA:2+TES5+XX11+200610:1438+00000001'
```

Comment:
Identical to `type2-first-mandatory.edifact.dat`?

Reply:
That was a test case added by @grzegorz-robak  but just renamed by me